pkg/api/types.go

```diff
@@ -513,7 +513,8 @@ type EnvVar struct {
 	// Required: This must be a C_IDENTIFIER.
 	Name string `json:"name"`
 	// Optional: no more than one of the following may be specified.
-	// Optional: Defaults to "".
+	// Optional: Defaults to ""; variable references ("$(VAR_NAME)") are
```

Comment:
FTR I am hands-off on this - just swamped. @a-robinson is running point

On Tue, May 19, 2015 at 11:17 AM, Paul Morie <notifications@github.com>
wrote:

> In pkg/api/types.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8149#discussion_r30629300>
> :
>
> > @@ -513,7 +513,8 @@ type EnvVar struct {
> >  	// Required: This must be a C_IDENTIFIER.
> >  	Name string `json:"name"`
> >  	// Optional: no more than one of the following may be specified.
> > -	// Optional: Defaults to "".
> > +	// Optional: Defaults to ""; variable references ("$(VAR_NAME)") are
>
> @derekwaynecarr <https://github.com/derekwaynecarr> if you like the copy
> here and in the v1 types.go file, I will change the other API versions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8149/files#r30629300>
> .
>